### PR TITLE
improve(optimistic-oracle-monitor): Make wei logs more readable

### PR DESCRIPTION
**Pull request by nicholaspai**
Signed-off-by: Nick Pai <npai.nyc@gmail.com>

**Motivation**

+ bonus feature adding emojis to OO proposer bot's error logs.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [X]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/monitors/src/OptimisticOracleContractMonitor.js [hunk 1]
```diff
@@ -27,6 +32,9 @@ class OptimisticOracleContractMonitor {
     this.lastDisputePriceBlockNumber = 0;
     this.lastSettlementBlockNumber = 0;
 
+    // Formats an 18 decimal point string with a define number of decimals and precision for use in message generation.
+    this.formatDecimalString = createFormatFunction(this.web3, 2, 4, false);
```
[the remark this answers is not in the document]
Comment (nicholaspai) [reply packages/monitors/src/OptimisticOracleContractMonitor.js:36]: Ah you're right, I can use this for price proposals which we assume will be 18 decimals but not for final fees.

[the remark this answers is not in the document]
Comment (nicholaspai) [reply packages/monitors/src/OptimisticOracleContractMonitor.js:38]: Done; had to add a helper method to grab the `collateralCurrency` for each requester contract. This extra async overhead is why I skipped this part originally :)

File: packages/core/contracts/oracle/implementation/test/OptimisticRequesterTest.sol [hunk 2]
```diff
@@ -17,6 +17,11 @@ contract OptimisticRequesterTest is OptimisticRequester {
     uint256 public refund;
     int256 public price;
 
+    // Implement collateralCurrency so that this contract
```
Comment (chrismaree) [inline packages/core/contracts/oracle/implementation/test/OptimisticRequesterTest.sol:20]: nit: line length. pretty sure you can fit this in 2 lines under 120 chars each.

File: packages/monitors/src/OptimisticOracleContractMonitor.js [hunk 3]
```diff
@@ -185,6 +201,16 @@ class OptimisticOracleContractMonitor {
     this.lastSettlementBlockNumber = this._getLastSeenBlockNumber(latestEvents);
   }
 
+  // Returns helper method for converting collateral token associated with financial contract
+  // to human readable form.
```
Comment (chrismaree) [inline packages/monitors/src/OptimisticOracleContractMonitor.js:205]: nit: line length.
```suggestion
 // Returns helper method for converting collateral token associated with financial contract to human readable form.
```

Comment (chrismaree) [reply packages/monitors/src/OptimisticOracleContractMonitor.js:205]: bit of a petty this requires two additional sequential async calls but not really any way around it.

File: packages/monitors/src/OptimisticOracleContractMonitor.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrismaree) [reply packages/monitors/src/OptimisticOracleContractMonitor.js:38]: looks good!

**Review by chrismaree — APPROVED** [review]
Thanks for adding the changes I requested! Looks good to me!

**Pull request merged**